contrib/pg_tde/src/pg_tde_guc.c

```diff
@@ -14,8 +14,6 @@
 #include "postgres.h"
 #include "utils/guc.h"
 
-#ifndef FRONTEND
```

Comment:
This seems possibly correct but should be part of a different PR or at least a different commit which explains why this is safe.

Reply:
I think I see what probably happened. You remove this instead of `PERCONA_EXT`.

Reply:
Yes, I removed the wrong definition, fixed.